### PR TITLE
[WIP] Organize errors, warnings and fixes during CGMES conversion

### PR DESCRIPTION
**Pull request by zamarrenolm**
Potential errors found during CGMES conversion process have been compiled and encoded in a single class. 

This will help in documenting current behaviour of the converter.

Errors and fixes may come from missing or invalid data in the input CGMES model, from limitations on the destination IIDM model, or from the required transformations.

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Errors.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class Errors {
```
Comment (MioRtia) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Errors.java]: I think `Warnings` would be a more accurate name than `Errors`

Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Errors.java]: Agree, changed

**Comment by MioRtia** [comment]
Could you please sort your Warnings by alphabetical order? This way, it will be easier to search through them.

**Comment by zamarrenolm** [comment]
> Could you please sort your Warnings by alphabetical order? This way, it will be easier to search through them.

Done

**Pull request closed without merging**